cfgov/sheerlike/templates.py

```diff
@@ -4,16 +4,9 @@
 
 
 def _convert_date(date, tz):
-    if date:
-        if isinstance(date, basestring):
-            date = parser.parse(date,
-                                default=datetime.datetime.today().replace(day=1))
-        if type(date) in [datetime.datetime, datetime.date]:
-            if isinstance(date, datetime.datetime):
-                pytzone = timezone(tz)
-                if date.tzinfo:
-                    date = date.replace(tzinfo=None)
-                date = pytzone.localize(date)
+    if date and isinstance(date, basestring):
+        date = parser.parse(date,
+                            default=datetime.datetime.today().replace(day=1))
     return date
 
```

Comment:
Thanks, I can confirm that everything is working as expected with the event livestream. @rosskarchner's change essentially does what I mentioned here, but you're right that `when` works without it. 

However, it's weird - it only works without it because somehow the page's event times being passed in to `when`, which are stored in UTC, are passed in as ET. Do you know why that might be? It's called in `event.html` and if you put in a debug statement in the first line of `when`, you'll see that `starttime` or `streamtime` are in ET.  Just curious; perhaps that's a feature of the whole TIME_ZONE setting.